### PR TITLE
Gus

### PR DESCRIPTION
**Pull request by jsvana**

### CODE REVIEW

**Comment by jsvana** [comment]
This is not for landing.

File: src/config.rs [hunk 1]
```diff
@@ -60,3 +60,7 @@ impl Config {
             .map_err(|e| format_err!("Failed to read configuration: {}", e))
     }
 }
+
```
Comment (guswynn) [inline src/config.rs:63]: single comment test

File: src/server.rs [hunk 2]
```diff
@@ -21,6 +21,10 @@ use super::irc::Message;
 
 pub type GuardedQueueMap = Arc<Mutex<BTreeMap<String, Sender<Message>>>>;
 
+fn yo() -> Result<()> {
+    Ok(())
+}
+
```
Comment (guswynn) [inline src/server.rs:27]: start a review on single comment

**Review by guswynn — CHANGES_REQUESTED** [review]

File: src/main.rs [hunk 3]
```diff
@@ -1,40 +1,20 @@
...
 use std::collections::BTreeMap;
-use std::net::ToSocketAddrs;
-use std::pin::Pin;
-use std::str::FromStr;
 use std::sync::Arc;
 
-use anyhow::{format_err, Result};
-use futures::channel::mpsc::{channel, Receiver, Sender};
-use futures::future::{join, join_all};
+use anyhow::Result;
```
Comment (guswynn) [inline src/main.rs:8]: review on all changes

**Review by guswynn — APPROVED** [review]

**Pull request closed without merging**